conda-recipe/meta.yaml

```diff
@@ -22,5 +22,12 @@ requirements:
     - python
     - mongoengine
     - six
-
-
+    - pyyaml
+
+test:
+  # Python imports
+  imports:
+    - metadataStore
+    - metadataStore.commands
+    - metadataStore.api
+    - metadataStore.conf
```

Comment:
Is there some reason that `.odm_templates` and `.document` are not included here?